src/groupmesh.cpp

```diff
@@ -149,7 +149,7 @@ void Group::GenerateForStepAndRepeat(T *steps, T *outs, Group::CombineAs forWhat
     std::vector<T> *scratch = &workA;
     // do the boolean operations
     while(n > 1) {
-#pragma omp parallel for
+//#pragma omp parallel for
```

Comment:
Should this be commented?

Reply:
It should probably be removed. Running that loop in parallel does work, but it ends up giving only 1 thread to each boolean operation which slows them down.